CollapseLauncher/XAMLs/MainApp/Pages/Dialogs/SimpleDialogs.cs

```diff
@@ -851,6 +855,28 @@ public static async Task<ContentDialogResult> Dialog_SteamShortcutCreationFailur
         }
         #endregion
 
+        internal static async Task<ContentDialogResult> Dialog_DownloadSettings(UIElement Content, GamePresetProperty currentGameProperty)
+        {
+            CheckBox checkBox = new CheckBox { Content = new TextBlock { Text = "Start game after install" }, IsChecked = currentGameProperty._GameInstall.StartAfterInstall };
+            checkBox.Click += (_, _) => currentGameProperty._GameInstall.StartAfterInstall = (bool)checkBox.IsChecked;
+
+            StackPanel panel = new StackPanel()
+            {
+                Orientation = Orientation.Vertical,
+                MaxWidth = 350,
```

Comment:
I don't think it's necessary to set the maximum width for the checkbox container as some localization might have a really long string for the checkbox since it doesn't support text wrapping.

Reply:
yeah, removed it